app/models/liabilitycalculation/viewmodels/TaxYearSummaryViewModel.scala

```diff
@@ -40,4 +42,27 @@ case class TaxYearSummaryViewModel(calculationSummary: Option[CalculationSummary
 
 }
 
+case class TYSClaimToAdjustViewModel(adjustPaymentsOnAccountFSEnabled: Boolean,
+                                     poaTaxYear: Option[TaxYear]) {
 
+  val claimToAdjustTaxYear: Option[TaxYear] = {
+    if (adjustPaymentsOnAccountFSEnabled) {
+      poaTaxYear
+    } else {
+      None
+    }
+  }
+
+}
+
+object TYSClaimToAdjustViewModel {
+
+  def ctaLink(isAgent: Boolean): String = {
+    if (isAgent) {
+      "/report-quarterly/income-and-expenses/view/agents/adjust-poa/start"
```

Comment:
Feel free to create required controller and routes to resolve this URL in the code